packages/manager/apps/pci-volume-backup/src/data/hooks/useVolume.spec.tsx

```diff
@@ -69,6 +69,7 @@ describe('useCreateVolumeFromBackup', () => {
     regionName: 'region-1',
     volumeBackupId: 'backup-1',
     volumeName: 'new-volume',
+    type: 'classic',
```

Comment:
Since classic is set to default type, maybe the check should be on anything else